gno.land/pkg/gnoweb/gnoweb_test.go

```diff
@@ -68,7 +69,8 @@ func TestRoutes(t *testing.T) {
 			app.Router.ServeHTTP(response, request)
 			assert.Equal(t, r.status, response.Code)
 
-			assert.Contains(t, response.Body.String(), r.substring)
+			body := response.Body.String()
+			assert.Contains(t, body, r.substring)
```

Comment:
Why the change?
I'd argue it's not more readable

Reply:
Didn't mean to commit that; used for debugging. Reverted